ui/src/components/Arrow.tsx

```diff
@@ -0,0 +1,41 @@
+export function arrow(angle: number): string {
```

Comment:
Can we make this func name a little more descriptive?

Reply:
This still needed?

Reply:
Yes but I updated it with the new styling